devops/containers/ubuntu2204_preinstalled.Dockerfile

```diff
@@ -3,12 +3,16 @@ ARG base_image=ghcr.io/intel/llvm/ubuntu2204_intel_drivers
 
 FROM $base_image:$base_tag
 
+USER ROOT
```

Comment:
does this even work with root in all-caps? anyway i would lowercase it :)

Reply:
Addressed in 6d88ec7e6da82651e4cd4fd6ceb3725d490511bb